### PR TITLE
Linear Search in JavaScript

### PR DESCRIPTION
**Pull request by jayantaadhikary**
Program to perform a linear search using Javascript.

### CODE REVIEW

File: Linear Search Algorithm/linear_search.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const linearSearch = (array, num) => {
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === num) {
+      return i;
+    }
+  }
+  return -1;
+};
+
+linearSearch(numbers, 7); //we call the function with the array and the number we want to search (7 in this case). It will return the index, ie, 2.
```
Comment (rishabh-live) [inline Linear Search Algorithm/linear_search.js]: numbers variable is not declared

**Review by rishabh-live — CHANGES_REQUESTED** [review]

File: Linear Search Algorithm/linear_search.js (hunk 1 above)
Comment (jayantaadhikary) [reply Linear Search Algorithm/linear_search.js]: Oh my bad. I changed the variable name at the last moment but forgot to change it while calling the function. Let me do it now.

**Comment by jayantaadhikary** [comment]
Is it okay now? Just corrected the silly mistake on my part.

**Review by rishabh-live — APPROVED** [review]
- [x] Approved

**Pull request merged**